src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml.cs

```diff
@@ -305,60 +270,22 @@ private void RefreshCondensedDisplay(Action refreshDisplay)
                 return;
             }
 
-            string newContent = "null";
+            string newContent = await GetCondensedStringAsync();
```

Comment:
As on Revit there is only one thread is running, changing to `await` doesn't work on Revit. Let's not to create two execution paths. :-)